magicgui/widgets/_function_gui.py

```diff
@@ -169,7 +188,9 @@ def __getattr__(self, value):
     #     """Delete a widget by integer or slice index."""
     #     raise AttributeError("can't delete items from a FunctionGui")
 
-    def __call__(self, *args: Any, **kwargs: Any):
+    _UNSET = object()  # sentinel used when temporarily overriding function.__globals__
```

Comment:
Can you elaborate on this comment?

Reply:
oops, sorry, I forgot about that one when i moved it down to the bottom of the file.  Removed

Reply:
Well, even so, if you're keeping _UNSET somewhere, can you explain why?

Reply:
oh sorry, that sentinel is used for this line:
```
original_value = function.__globals__.get(func_name, _UNSET)
```
where we check to see what object the function name points to in the module in which it was defined.  obviously, it will _normally_ just be pointing to the function object itself, but if for some reason it is not, we want to reset it to that value when we're done. aaand, if it's not set at all (perhaps someone used `del`), then we use the `_UNSET` as a placeholder (instead of `None`)